packages/core/src/physics/PhysicsManager.ts

```diff
@@ -0,0 +1,228 @@
+import { HitResult } from "./HitResult";
+import { Ray } from "@oasis-engine/math";
+import { IPhysics, IPhysicsManager } from "@oasis-engine/design";
+import { Collider } from "./Collider";
+import { Layer } from "../Layer";
+import { ColliderShape } from "./shape/ColliderShape";
+
+/**
+ * Filtering flags for scene queries.
+ */
+export enum QueryFlag {
+  STATIC = 1 << 0,
+  DYNAMIC = 1 << 1,
+  ANY_HIT = 1 << 4,
+  NO_BLOCK = 1 << 5
+}
```

Comment:
`QueryFlag`  may just be implemented internally. You can write this in `PhysXPhysicsManager`, The public interface for developer is `Layer` currently.

Reply:
done